resources/scripts/routers/DashboardRouter.tsx

```diff
@@ -9,6 +10,10 @@ import AccountApiContainer from '@/components/dashboard/AccountApiContainer';
 import NotFound from '@/components/screens/NotFound';
 
 export default ({ location }: RouteComponentProps) => (
```

Comment:
What the heck is this formatting. 🤔 

Should be
```
=> {

useEffect(...);

return (<div/>);
```

Reply:
Changed